po/fr.po

```diff
@@ -4412,7 +4412,7 @@ msgid ""
 msgstr ""
 "Tester la correspondance des expressions régulières sur tout le tampon d'un "
 "coup plutôt que ligne à ligne, ce qui permet aux correspondances de "
-"recouvrir plusieurs lignes.  Dans ce mode les caractères de fin de ligne "
+"recouvrir plusieurs lignes.  Dans ce mode, les caractères de fin de ligne "
```

Comment:
_"Le complément circonstanciel est obligatoirement suivi d’une virgule."_ could have been my argument, but, to be honest, it was just in reference to the English sentence L4409 :)